webapp/i18n/ja.json

```diff
@@ -2012,6 +2054,7 @@
   "user.settings.security.ldap": "AD/LDAP",
   "user.settings.security.loginGitlab": "GitLabでログインしました",
   "user.settings.security.loginLdap": "AD/LDAPでログインする",
+  "user.settings.security.loginSaml": "GitLabでログインしました",
```

Comment:
In translation server, this message is FUZZY and incorrect suggestion from `system user` ([unit330550](http://translate.mattermost.com/ja/mattermost/translate/web_static.po#unit=330550)).

I think that FUZZY messages shouldn't be included in PR, because suggestions from `system user` often has incorrectness.
Could you reject FUZZY messages when making PR?

(Of course, I'll translate these FUZZY messages on translation server later :D )